<launch>  
  <param name="/use_sim_time" value="true"/>
  <param name="leader_num" value="2"/>
  <param name="number_of_robots" value="3"/>
  
  <node name="stage" pkg="stage_ros" type="stageros" args="$(find formation)/world/willow-multi-erratic.world"/>  
  
  <node name="monitor" pkg="formation" type="monitor.py" output="screen"/>

  <include file="$(find formation)/launch/robot_0.launch"/>
  <include file="$(find formation)/launch/robot_1.launch"/>
  <include file="$(find formation)/launch/robot_2.launch"/> 
  <!--include file="$(find formation)/launch/robot_3.launch" /--> 
</launch>

